<launch>

  <arg name="bag" default="duo3d_frentebaixo"/>
  <param name="/use_sim_time" value="true"/>

  <!--descomprimir as imagens-->
  <node pkg="image_transport" type="republish" name="republish_left"  output="screen" args="compressed in:=/duo3d/left/image_rect raw out:=/duo3d/left/image_rect"/>
  <node pkg="image_transport" type="republish" name="republish_right" output="screen" args="compressed in:=/duo3d/right/image_rect raw out:=/duo3d/right/image_rect"/>

  <!--tocar o bag com taxa mais lenta-->
  <node pkg="rosbag" type="play" name="rosbag_play" output="screen" args="/home/$(env USER)/Desktop/$(arg bag).bag --clock -r 0.1"/>

</launch>
